### DISCUSSION TITLE
404 Layout on beta 51

### DISCUSSION THREAD

**Discussion by digital-codes** [vuepress/core#1075] [Q&A] [answered]
Beta 51 changed custom layout handling. I copied the default "NotFound.vue" to my docs folder (same as for Layout.vue) which works, but I cannot get rid of the message "Page does not exist. This is a fallback content."  on the page, see screenshot below

NotFound.vue:

```
<script setup lang="ts">
import { useRouteLocale } from '@vuepress/client'
import { useThemeLocaleData } from '@vuepress/theme-default/client'

import ParentLayout from '@vuepress/theme-default/layouts/Layout.vue'

const routeLocale = useRouteLocale()

const themeLocale  = useThemeLocaleData()

const homeLink = themeLocale.value.home ?? routeLocale.value
const homeText = themeLocale.value.backToHome ?? 'Back to home'
const messages = themeLocale.value.notFound ?? ['Not Found']
const getMsg = (): string =>
  messages[Math.floor(Math.random() * messages.length)]

</script>
<template>
  <ParentLayout>
    <template #page-content-top>
        <h1>404</h1>
        <blockquote>{{ getMsg() }}</blockquote>
        <RouterLink :to="homeLink">{{ homeText }}</RouterLink>
    </template>
  </ParentLayout>
</template>

```

Any ideas how to remove that message ?

![Bildschirmfoto am 2022-09-05 um 20 06 36](https://user-images.githubusercontent.com/34965919/188498720-4e816435-ddfa-4063-b8a4-c528c551edfb.png)

**Comment by Mister-Hope** [comment]
You may provide a mininal reproducion and open a bug report at issue panel. This is not the place for you to report problems with some simple lines.

**Comment by meteorlxy** [comment, marked as answer]
The `NotFound` layout should not use the `<Content />` component.

You are using the `Layout` layout of default theme as the `NotFound` layout, which includes the `<Content />` component for sure...
